lib/send-text.js

```diff
@@ -0,0 +1,15 @@
+// Copyright IBM Corp. 2016. All Rights Reserved.
+// Node module: strong-error-handler
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+var format = require('util').format;
+
+module.exports = function sendText(res, data) {
+  var content = format('%s: %s [%s]', data.name, data.message,
+    data.statusCode);
```

Comment:
This is a good start 👍 

I am concerned that this format is missing important additional information, e.g. `err.details` from validation errors (in both production and debug model), additional fields like `stack` in debug mode.

Please check how the built-in [html renderer](https://github.com/strongloop/strong-error-handler/blob/4c69370751755c0eb1a96e30a7f62182704839ba/views/default-error.ejs) is iterating over error properties and implement a similar algorithm here.

Reply:
👍